src/Resources/Product.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Tighten\NovaStripe\Resources;
+
+use Illuminate\Support\Str;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\ID;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Http\Requests\NovaRequest;
+
+class Product extends BaseResource
+{
+    public static $model = \Tighten\NovaStripe\Models\Product::class;
+
+    public static $title = 'name';
+
+    public static $search = [
+        'id',
+        'name',
+    ];
+
+    public static function uriKey()
+    {
+        return 'stripe-products';
+    }
+
+    public function fields(NovaRequest $request)
+    {
+        return [
+            ID::make()
+                ->hideFromIndex(),
+
+            Text::make('Name')
+                ->sortable(),
+
+            Text::make('Price')
+                ->sortable()
+                ->displayUsing(function () {
+                    $currency = Str::upper($this->default_price['currency']) ?? '';
+
+                    return $currency . ' ' . number_format($this->default_price['unit_amount'] / 100, 2);
+                }),
+
+            Text::make('Type')
+                ->displayUsing(fn () => Str::of($this->default_price['type'])->replace('_', ' ')->title()),
```

Comment:
Similar to the previous, what do you think?
```suggestion
                ->displayUsing(fn() => Str::of(data_get($this->default_price, 'type', ''))->replace('_', ' ')->title()),
```

Reply:
Done!